sources/packages/backend/libs/test-utils/src/factories/user.ts

```diff
@@ -1,9 +1,9 @@
 import * as faker from "faker";
 import { User } from "@sims/sims-db";
 
-export function createFakeUser(): User {
+export function createFakeUser(userName?: string): User {
   const user = new User();
-  user.userName = faker.random.uuid();
+  user.userName = userName ?? faker.random.uuid();
```

Comment:
what is the possibility of random uuid to be acceptable?

Reply:
if the method doesn't get a username as a parameter. then the test will create a fake username